format/format.go

```diff
@@ -0,0 +1,74 @@
+package format
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"unicode/utf8"
+
+	"github.com/riraum/hn-cli/http"
+	"github.com/riraum/hn-cli/item"
+)
+
+func Format(frontpageIDs []int, tWidth int) (item.Item, error) {
```

Comment:
I would actually change that to use `[]item.Item` instead of loading the items in this function. maybe for a future PR?

Reply:
fair, I'd still argue that `Format` should only "format" the items, so maybe split this logic in a future PR?